pkg/controller/operconfig/ingressconfig_controller.go

```diff
@@ -0,0 +1,115 @@
+package operconfig
+
+import (
+	"context"
+	"log"
+
+	operv1 "github.com/openshift/api/operator/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+// newIngressConfigReconciler returns a new reconcile.Reconciler
+func newIngressConfigReconciler(client client.Client) *ReconcileIngressConfigs {
+	return &ReconcileIngressConfigs{client: client}
+}
+
+var _ reconcile.Reconciler = &ReconcileIngressConfigs{}
+
+// ReconcileIngressConfigs watches for updates to ingress controller configuration
+// and sets the network policy related labels on the openshift-host-network namespace
+type ReconcileIngressConfigs struct {
+	client client.Client
+}
+
+// constants for namespace and custom resource names
+// namespace in which ingress controller objects are created
+const IngressControllerNamespace = "openshift-ingress-operator"
+
+// namespace representing host network traffic
+// this is also the namespace where to set the ingress label
+const HostNetworkNamespace = "openshift-host-network"
```

Comment:
probably should have this defined in `pkg/names`; we'll need to write it out in the ovn-kubernetes configmap later too

Reply:
done.